docs/topics/forms/formsets.txt

```diff
@@ -46,6 +52,33 @@ Formsets can also be indexed into, which returns the corresponding form. If you
 override ``__iter__``, you will need to also override ``__getitem__`` to have
 matching behavior.
 
+Method 2: Using ``FormSet`` class
+---------------------------------
+
+.. class:: FormSet
+
+You can use the ``FormSet`` class to declare a formset in same way you declare models.
+This is alternative method to using ``formset_factory`` to make the formset class.
+
+To create a formset out of an ``ArticleForm`` you would do::
+
+    >>> from django.forms import formsets
+    >>> class ArticleFormSet(formsets.FormSet):
+    ...     form = ArticleForm
+
+.. note::
+    It is compulsory to pass the form argument while declaring the Formset class. Similar to
+    how it's compulsory to pass form as first argument to ``formset_factory``.
```

Comment:
Is a suitable error raised if I forget to do this?

Reply:
As of now, there is no error is raised.
I tried that, but it always raises an error by default when the instance of ModelFormSet is created in django.forms.models. Even if there is a simple warning it will be trigged every time. So I didn't put any warning or error there